instance/models/instance.py

```diff
@@ -410,6 +410,27 @@ def ansible_swift_settings(self):
                                 'region': self.swift_openstack_region})
 
     @property
+    def lms_preview_sub_domain(self):
+        """
+        LMS preview sub-domain name (eg. 'preview.master')
+        """
+        return 'preview.{}'.format(self.sub_domain)
```

Comment:
@bradenmacdonald Ah, sorry about that, it totally slipped my mind. I will change it, and I'll also change the studio subdomain in this same PR.